SobokSobok/SobokSobok/Presentation/Medicine/AddMyMedicineView/AddMyMedicineViewController.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  AddMyMedicineViewController.swift
+//  SobokSobok
+//
+//  Created by 김승찬 on 2022/01/10.
+//
+
+import UIKit
+
+import EasyKit
+
+final class AddMyMedicineViewController: BaseViewController {
+    
+    // MARK: Properties
+    
+    var medicineData: [String] = []
+    let addMyMedicineView = AddMyMedicineView()
+    
+    override func loadView() {
+        self.view = addMyMedicineView
+    }
+    
+    // MARK: View Life Cycle
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setBackgroundColor()
+        assignDelegate()
+        collectionViewReload()
+        actionPeopleSelectButton()
+    }
+    
+    // MARK: Functions
+    
+    private func setBackgroundColor() {
+        view.backgroundColor = .white
+    }
```

Comment:
```suggestion
override func style() {
    view.backgroundColor = .white
}
```
스타일(UI) 관련 코드는 BaseViewController의 style 함수를 오버라이딩해서 코드를 작성해주시면 좋을 것 같습니다. (viewDidLoad에서 호출안해도됨.)